pkg/api/api.go

```diff
@@ -22,3 +22,14 @@ const (
 	OpenShiftConsoleRouteName         = OpenShiftConsoleShortName
 	OAuthClientName                   = OpenShiftConsoleName
 )
+
+// configs always have the name "cluster"
```

Comment:
`cluster` is the correct name for our configs, however. 

- `config.openshift.io`, `Console`, name: `cluster`
- `operator.openshift.io`, `Console`, name: `cluster`